cloud_info_provider/utils.py

```diff
@@ -29,3 +30,8 @@ def get_tag_value(xml, tag):
         return xml.getElementsByTagName(tag)[0].firstChild.nodeValue
     else:
         return None
+
+
+def pythonize_network_info(network_info):
+    '''Pythonize network_info string'''
+    return ast.literal_eval(network_info.replace(':"', '"').replace("=>", ":"))
```

Comment:
You are right, if it's supposed to be expressed as JSON it would be cleaner to use a JSON tool.
As you have seen the output was a bit garbish. Let's clarify this with cloudkeeper guys and see what to do.

Reply:
So it's now using `json.loads` instead of `ast.literal_eval`. Still the format is not json and weirdy.